hypothesis_sqlalchemy/core/column_type.py

```diff
@@ -184,6 +186,17 @@ def _(type_: postgresql.UUID,
     return result
 
 
+json = strategies.recursive(
+    strategies.none() | strategies.booleans() | strategies.floats() | strategies.text(printable),
```

Comment:
can it be any UTF-8 string? why are we limiting ourselves only with printable ASCII characters?

Reply:
hey, thanks for your review 👋 

i think it might be better to per-user overload the `scalars` function. that is what i did in the end for my case.

the need for json type could be too specific.

wonderful lib btw!

Reply:
we can also pass custom strategies for columns through variadic keyword-only parameters (aka `**kwargs`), like `email_address` in [the example from README](https://github.com/lycantropos/hypothesis_sqlalchemy#records)